app/views/assignments/peer_review.html.erb

```diff
@@ -0,0 +1,108 @@
+<div class='title_bar'>
+  <h1>
+    <%= t('student.peer_assignment',
+          short_identifier: @assignment.short_identifier) %>
+  </h1>
+</div>
+
+<div class='student_columns'>
+  <div id='student_column_left'>
+    <div class='block'>
+      <%= render partial: 'read', locals: { peer_review: true } %>
+    </div>
+
+    <div class='block'>
+      <h2><%= t('submissions') %></h2>
+
+      <div class='block_content'>
+        <% unless PeerReview.find_by(reviewer_id: @student.id).nil? %>
```

Comment:
@GitGeminiano This should be the grouping id, not the student id. Just tried to view the page with a fresh database, and couldn't see any tables appear.